dash/dash.py

```diff
@@ -573,6 +573,8 @@ def __init__(  # pylint: disable=too-many-statements
             for plugin in plugins:
                 plugin.plug(self)
 
+        self._setup_hooks()
+
         # tracks internally if a function already handled at least one request.
```

Comment:
why do we need/want this?

Reply:
Yes, it's the setup that load the plugins and add the setup/callback/error hooks.